src/main/java/com/palantir/common/streams/KeyedStream.java

```diff
@@ -148,6 +140,27 @@
             BiFunction<? super K, ? super V, ? extends Stream<? extends Map.Entry<? extends K2, ? extends V2>>> entryMapper);
 
     /**
+     * Performs an action for each key in the stream.
+     */
+    default void forEachKey(Consumer<? super K> action) {
+        forEach((key, value) -> action.accept(key));
```

Comment:
Would you mind inlining this? It's very little complexity saved, and it makes debugging easier if there aren't two extra stack frames to step through